packages/cli/src/ui/components/ConfigInitDisplay.tsx

```diff
@@ -31,9 +31,9 @@ export const ConfigInitDisplay = () => {
       setMessage(`Connecting to MCP servers... (${connected}/${clients.size})`);
     };
 
-    appEvents.on('mcp-client-update', onChange);
+    appEvents.on(AppEvent.McpClientUpdate, onChange);
     return () => {
-      appEvents.off('mcp-client-update', onChange);
+      appEvents.off(AppEvent.McpClientUpdate, onChange);
     };
   }, [config]);
```

Comment:
optional nit: not a new issue but a bit odd that this useEffect is based on config but doesn't use config at all within the effect. Consider refactoring and cleaning this up in the future.